core/src/main/java/org/stellar/anchor/filter/PlatformAuthJwtFilter.java

```diff
@@ -0,0 +1,19 @@
+package org.stellar.anchor.filter;
+
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import org.stellar.anchor.auth.ApiAuthJwt.PlatformAuthJwt;
+import org.stellar.anchor.auth.JwtService;
+
+public class PlatformAuthJwtFilter extends AbstractJwtFilter {
+  public PlatformAuthJwtFilter(JwtService jwtService) {
+    super(jwtService);
+  }
+
+  @Override
+  void check(String jwtCipher, HttpServletRequest request, ServletResponse servletResponse)
+      throws Exception {
+    PlatformAuthJwt token = jwtService.decode(jwtCipher, PlatformAuthJwt.class);
```

Comment:
We check that `token` is not null in `Sep10JwtFilter`. Do we need a check here as well?

Reply:
No, token is checked later in the flow. Checking it here will disable support for auth token

Reply:
The decode function never returns `null`. I have changed the variable type to `@NonNull`.